### ISSUE TITLE
Processing reader conditionals broken

### ISSUE THREAD

**Issue by metasoarous** [clj-commons/marginalia#164]
I see from #103 that `cljc` should now be fixed now. However, I'm still not able to generate documentation for https://github.com/metasoarous/semantic-csv (try executing the `./build_docs.sh` script), and get the following error:

```
Exception in thread "main" java.lang.RuntimeException: Problem parsing near line 1 <(:clj [clojure.java.io :as io])> original reported cause is java.lang.RuntimeException: Conditional read not allowed -- java.lang.RuntimeException: Conditional read not allowed, compiling:(/tmp/form-init9194388453733717094.clj:1:73)
        at clojure.lang.Compiler.load(Compiler.java:7442)
        ...
```

Any updates on this?

**Comment by gdeer81** [comment]
Thanks for reporting, I'll put this on the roadmap for 9.2

**Comment by gdeer81** [comment]
This one is driving me nuts, I added a cljc file to the test projects and I can't seem to reproduce the error locally with lein test.

I can reproduce it by cloning semantic-csv and running lein marge on it, but that's not really something I can add to my automated tests

**Comment by metasoarous** [comment]
Woah; crazy. Do you have a branch for this? I could try to take a look and see if I can't get your test to fail.

Do you have a hunch on the issue? Could clj/cljs version discrepancy be at fault, or do you think there's some edge case not being caught?

Thanks for your time looking into this. I've been wanting to mint a new release of semantic-csv, but have been stalling to see if we can't get the docs up in order.

**Comment by gdeer81** [comment]
I think it's because my automated tests are broken so the code in the test projects folder is being missed somehow. Once I shave that yack I should be able to get back to the issue at hand.

The quick fix might be to just move my parser code into a cljc file so if the reader thinks it's running the code in the parser context it will be fine.

**Comment by gnl** [comment]
Any updates on or workarounds for this?

**Comment by metasoarous** [comment]
@gdeer81 Has there been any progress on this? I've been waiting to release semantic-csv out of alpha for about a year now because I don't want to do so without having updated the docs. I really like marginalia and don't want to switch doc systems if I don't have to, but my users are getting a bit restless at not having a recent stable & documented release out.

**Comment by darren-fhf** [comment]
I've been seeing this same error. 

When I built and installed darwindarak's pull request #175 it fixed the problem. +1 to that PR.

**Comment by metasoarous** [comment]
Awesome! Thanks for the update @darren-fhf.

@gdeer81 Can we please get this merged in and released? I'd been holding off on minting a new release of semantic-csv till the docs are sorted out, and would love to get it out!

Thanks in advance!

**Comment by tsmacdonald** [comment]
Fixed in #183; will be released in 0.9.2 (give me a ping if you're antsy for the release to come out).
